app-crypt/acme-dns/Manifest

```diff
@@ -0,0 +1,5 @@
+AUX acme-dns.conf 81 BLAKE2B 26d0d6f0c64e0728f611e1b63a07e11ebb7f090302f012fde419eb057c86298599985f938ee1680e7f7b518cf3d395338507a8efb055ecd758ade768fa98cea0 SHA512 4ac59e33c472f82799df88d8d26e4601ff82df347419639357ef2db4d4dc0cf719499abddb08191a1fe76d274b78d24af603e5a9c7978ef5ad897ea6aaa6d4b9
+AUX acme-dns.init 487 BLAKE2B 107fb2165218afe4dd4960be36648cb7fc31030501e204ce68adff71f50843299eeff2db4138689227c2c6d70b89097095e31c93bb78abe6de842b0c279ba537 SHA512 d47d8fb4f1383b48cb8246695f87552154b8a0998820739158fd097d566fde96544ad51dfbb933125b8082b3abc5655dc388bc3bf2ee3d37262b800d31b07900
+DIST acme-dns-0.4.tar.gz 14265221 BLAKE2B 5b3b075a15cc7e4d145815c09627c7d8697c7393a53a34022993f240887b4dd7f210a6fc7f7b58bfab2cfa611f8dfd4b7f053e17f2576693fe843fa346ba39ae SHA512 a5b0e708cd1d8efbf8ea329d3ecd7221a5e73b7c5bae6fca316cb08a12bc661fed1130e205d148a23364e1b8f8dc539090c20796fbcbf8d09f083c86ca9774aa
+EBUILD acme-dns-0.4.ebuild 1710 BLAKE2B dbbe009205788cbb8a8462ff8ebb02292ace5e0b18b39219932422ffe1135fc73788b729072f1e5b70262f8b63a3134a6b44c8c8716a5fdb945753e945c668b5 SHA512 15d7c5fbd4f5aea516c188d218b1125d1d3bbb9c72c99660986201d368b3789b7834ae82e9d6159e43369f694147eff632c1a9016fabaa54dc491777bb7f6fbc
```

Comment:
I believe these were not generated using `repoman manifest`? Because I don't think it would generate EBUILD and metadata hashes here.

Reply:
Nope, `ebuild ... digest`. Shall I redo with repoman?

Reply:
I'm sure @Dr-Terrible can give us an explanation here on the difference. But yes, using `repoman` would be better since that's what our CI does.

Reply:
Hmm, `repoman manifest` produces exactly the same Manifest.

Reply:
Hmm, this is the result of `repoman manifest` here:

```
diff --git a/app-crypt/acme-dns/Manifest b/app-crypt/acme-dns/Manifest
index 1faf03a..f9fcb0d 100644
--- a/app-crypt/acme-dns/Manifest
+++ b/app-crypt/acme-dns/Manifest
@@ -1,5 +1 @@
-AUX acme-dns.conf 81 BLAKE2B 26d0d6f0c64e0728f611e1b63a07e11ebb7f090302f012fde419eb057c86298599985f938ee1680e7f7b518cf3d395338507a8efb055ecd758ade768fa98cea0 SHA512 4ac59e33c472f82799df88d8d26e4601ff82df347419639357ef2db4d4dc0cf719499abddb08191a1fe76d274b78d24af603e5a9c7978ef5ad897ea6aaa6d4b9
-AUX acme-dns.init 487 BLAKE2B 107fb2165218afe4dd4960be36648cb7fc31030501e204ce68adff71f50843299eeff2db4138689227c2c6d70b89097095e31c93bb78abe6de842b0c279ba537 SHA512 d47d8fb4f1383b48cb8246695f87552154b8a0998820739158fd097d566fde96544ad51dfbb933125b8082b3abc5655dc388bc3bf2ee3d37262b800d31b07900
 DIST acme-dns-0.4.tar.gz 14265221 BLAKE2B 5b3b075a15cc7e4d145815c09627c7d8697c7393a53a34022993f240887b4dd7f210a6fc7f7b58bfab2cfa611f8dfd4b7f053e17f2576693fe843fa346ba39ae SHA512 a5b0e708cd1d8efbf8ea329d3ecd7221a5e73b7c5bae6fca316cb08a12bc661fed1130e205d148a23364e1b8f8dc539090c20796fbcbf8d09f083c86ca9774aa
-EBUILD acme-dns-0.4.ebuild 1710 BLAKE2B dbbe009205788cbb8a8462ff8ebb02292ace5e0b18b39219932422ffe1135fc73788b729072f1e5b70262f8b63a3134a6b44c8c8716a5fdb945753e945c668b5 SHA512 15d7c5fbd4f5aea516c188d218b1125d1d3bbb9c72c99660986201d368b3789b7834ae82e9d6159e43369f694147eff632c1a9016fabaa54dc491777bb7f6fbc
-MISC metadata.xml 314 BLAKE2B fd3aa48b274a10b3467a3ee9c1d32d055b89542ec234b4e8cb76611e5a06bc0c60889ecf64e4606bf43b532733c792468a4a9cda9c5a8313f54d229eb801a49c SHA512 f12bffd4c91650e42e121d636e00d726fe0068843d0e3ae67f485eacdb0121bb98b8e4d4ec290bcf545c57d617619637aed74f1b10884ca9eafad81cb62ba6d8
```

Reply:
> Hmm, repoman manifest produces exactly the same Manifest.

The AUX + MISC checksum is the old Manifest format. The DIST-only checksum is the new format and is called Thin Manifest: https://wiki.gentoo.org/wiki/Repository_format/package/Manifest
https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf#thin-manifests

Thin manifests are explicitly enabled for this repository (see metadata/layout.conf), so I'm not sure what is happening here. Maybe @svoop's system is missing some critical update; what about trying: `emerge -auDNv @world @system --with-bdeps y`

Reply:
@Dr-Terrible As for the world upate: My box has no pending updates, portage is at 2.3.24-r1:0 and repoman at 2.3.6:0.

Reply:
@bpinto Good to know `repoman` has to be run in the context of the entire overlay since I usually develop with a local overlay as well. The Manifest looks the way it should now, fixup into d65f2ba